packages/atomic/src/components/common/atomic-modal/atomic-modal.tsx

```diff
@@ -54,6 +54,10 @@ export class AtomicModal implements InitializableComponent<AnyBindings> {
   @Prop({reflect: true, mutable: true}) isOpen = false;
   @Prop({mutable: true}) close: () => void = () => (this.isOpen = false);
   @Prop({reflect: true}) noFocusTrap = false;
+  /**
+   * Whether to display the open and close animations over the entire page or the atomic-modal only.
+   */
+  @Prop({reflect: true}) boundaries: 'page' | 'element' = 'page';
```

Comment:
Wonder if we should not use singular form here ? `page boundary` or `element boundary` VS `page boundaries` or `element boundaries` ... 🤔

Reply:
Looks like the correct word is "boundary", TIL